axiom_i2c_comms.c

```diff
@@ -29,6 +29,7 @@
 #include <linux/pm.h>
 #include <linux/i2c.h>
 #include <linux/string.h>
+#include <linux/version.h>
```

Comment:
This include is not required, as it has already been introduced with 6444ee6

Reply:
I have removed the duplicated include.